services/loadbalancer/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## v0.XX.X (2024-XX-XX)
+## v0.13.1 (2024-07-10)
```

Comment:
I'm not sure about this being a patch version, if the user has a linter like we do the builds will start failing.

I think this should be minor, since in the "Go way" we are only now effectively deprecating the methods

Reply:
Good point